apps/evm/test/support/evm_test_runner.ex

```diff
@@ -0,0 +1,110 @@
+defmodule EvmTestRunner do
+  import ExthCrypto.Math, only: [hex_to_bin: 1, hex_to_int: 1]
+
+  alias EVM.Mock.{MockAccountRepo, MockBlockHeaderInfo}
+
+  def run(json) do
```

Comment:
Not sure if this is desired, but I _think_ the other test runners accept a test name and retrieve the file. We could do that to keep in line with them, and it makes it easier to reuse the test runner with a rake task when we only wan to run a single test. What do you think?

Reply:
I like the idea of doing this for the consistency across test runners. I would like to keep reading files out of the is test_runner to eliminate it as a factor when benchmarking/running profiling. How would you feel if I opened a PR to remove reading files from the other test_runners?